specifications/api/swagger.yaml

```diff
@@ -1700,6 +1728,25 @@ components:
       properties:
         settings: *settings
 
+    ApiSmashServer: &ApiSmashServer
+      type: string
+      pattern: '^https?:\/\/[a-zA-Z0-9-_~.]+(:[0-9]+)?/?$'
+      example: https://smash.cardano-mainnet.iohk.io/
+      description: A base SMASH uri without endpoint path.
+
+    ApiHealthCheck: &ApiHealthCheck
+      type: string
+      enum: ["available", "unavailable", "unreachable", "no_smash_configured"]
+      description: |
+        The status of the SMASH server. Possible values are:
+
+        status                  | description
+        ---                     | ---
+        `"available"`           | server is awaiting your requests
+        `"unavailable"`         | server is running, but currently unavailable, try again in a short time
+        `"unreachable"`         | server could not be reached or didn't return a health status
+        `"no_smash_configured"` | SMASH is currently not configured, adjust the Settings first
```

Comment:
The health check status as a raw string feels a bit rough and prevent any further extension. What about wrapping this one in a `{ "health": ... }` or `{ "smash": ... }` or `{ "status": ... }` singleton..